app/views/items/_new_bid.html.erb

```diff
@@ -1,4 +1,10 @@
 <%= form_for ([item, @bid]) do |f| %>
-  <%= f.text_field :price  %>
+  <%= f.number_field :price, min: (item.high_bid+1.00) %>
   <%= f.submit "Place Bid"  %>
 <% end  %>
+<br />
+<br />
+<br />
+<br />
```

Comment:
The field has a popup error message that displays below it when a user enters the wrong input. But the field is at the bottom of the page, and the error message was not visible because it's pushed down below the bottom of the screen. These extra lines are just a quick way to make some extra space. They should be taken out when we adjust the layout.